test/get.allow.token.test.ts

```diff
@@ -0,0 +1,24 @@
+import { RelayingServices } from '../src';
```

Comment:
for me it's still not clear what the difference is (conceptually) between this file and `test/allow.token.test.ts`.

aren't they both testing the allow token functionality?

Reply:
I think that there are 2 separate functions and need separate test.